packages/core/src/org.ts

```diff
@@ -31,16 +31,27 @@ export enum OrgRole {
      * Member information (like public key and email address) of suspended members
      * are considered unverified, and need to be updated and verified via a
      * membership confirmation [[Invite]].
+     * @deprecated Use `OrgMemberStatus.Suspended` instead
      */
     Suspended,
 }
 
+export enum OrgMemberStatus {
+    Provisioned = "provisioned",
+    Active = "active",
+    Suspended = "suspended",
+}
+
 /**
  * Represents an [[Account]]s membership to an [[Org]]
  */
 export class OrgMember extends Serializable {
+    get id() {
+        return this.accountId;
+    }
+
     /** id of the corresponding [[Account]] */
-    id: AccountID = "";
+    accountId?: AccountID = undefined;
```

Comment:
Organization members are no longer guaranteed to have a corresponding account (yet), so we'll use the email address as a primary way of reference now.